package.json

```diff
@@ -52,7 +52,12 @@
     "argparse": "^1.0.3",
     "jscs": "^1.13.1",
     "rollup": "^0.36.3",
+    "rollup-plugin-commonjs": "^5.0.5",
+    "rollup-plugin-node-resolve": "^2.0.0",
     "rollup-watch": "^2.5.0",
     "uglify-js": "^2.6.0"
+  },
+  "dependencies": {
+    "omnitone": "^0.1.6"
```

Comment:
This makes me nervous. This is a wider conversation we should have, but I feel like being more permissive than `~` on our node modules is asking for trouble.